cutevariant/gui/viewquerywidget.py

```diff
@@ -323,7 +341,9 @@ def query(self, query: Query):
 
     def updateInfo(self):
 
-        self.page_info.setText(f"{self.model.total} variant(s)")
+        self.page_info.setText(
+            self.tr("{}-{} of {} variant(s)").format(*self.model.displayed())
```

Comment:
It's not readable to have "0-50 of 73" between page selector. And I plan to make the page number editable. 
We can move this information somewhere else in the bar. ( Left side ) 
For example:

     50 shown of 73 variants      <  3 of 40 >

Reply:
Ok i will change that.
In fact actual inspiration was SQLite Database Browser

Reply:
I tested it ! And finally it's readable ! Thanks for that!  We will see later how to manage page selection